### ISSUE TITLE
is metalsmith dead ?

### ISSUE THREAD

**Issue by leviwheatcroft** [metalsmith/metalsmith.io#236]
Obviously the project isn't dead, but the question has come up on the slack channel 3 times in the last 5 days. I guess that's a fair indication that many more people peruse metalsmith.io and wonder the same thing. Who knows how many would choose another project which happens to be a little bit more new and shiny.

As to how to address this on metalsmith.io, my suggestion would be to include a chart of package downloads.. it's in keeping with the terse nature of metalsmith.io, the download trend really does speak for itself, and while not relevant directly to metalsmith, I like that it uses dynamic content on a static site.

I'd be happy to have a go at implementing this but wanted to see if maintainers think it's a reasonable approach before I get started.

I've made a rad [demo of what I'm talking about](http://codepen.io/leviwheatcroft/pen/zZNadq). Basically just use npmjs no-auth JSON downloads api, and render the data with chartistjs.

**Comment by Ajedi32** [comment]
Regarding the question in the title, see segmentio/metalsmith#243

TL;DR: While there hasn't been a lot of development on core lately (for various reasons), Metalsmith certainly isn't dead, and if changes to the core were necessary they'd be pretty easy to make.

And yeah, if you can find a way to include a graph like that on the site without distracting too much from other, more important content (such as the introduction to Metalsmith and explanation of how it's used) that'd be fine with me.

**Comment by leviwheatcroft** [comment]
I'm not going to have time to go ahead with this. 

In addition, i have concerns about messing up a nice, fast, firm, experience on the page with an over engineered clunky graph.

Maybe its a good solution, but that's up to someone else to decide.. 

@axe312ger interested?

**Comment by axe312ger** [comment]
Basically yeah, time issue are another thing :D

How about a call-to-action button:

[:rocket: Metalsmith is rising]

Or sth like this? Maybe I can catch one of our UX designers, but they are suuuper busy ^^

**Comment by leviwheatcroft** [comment]
Maybe wait and see what comes of @woody's hangout.. there might be other changes to the site.

I feel like maybe we've out grown a readme style single page, but that's a separate issue.

**Comment by axe312ger** [comment]
Valid point! :)

**Comment by woodyrew** [comment]
This discussion has been happening on the Slack channel also, we organised a _meeting_ to discuss how we could make the perception that the Metalsmith project is not dead. The consensus being that the metalsmith.io site would go a long way to answering peoples common questions, getting started, and convey that the project's adoption is steadily increasing over time.

I have added the following users as collaborators on this repo to help to this end.

- @axe312ger
- @tsdorsey
- @leviwheatcroft
- @darisi

**Please branch and create pull requests rather than committing directly on master.** It would also be good for at least one other person to review before merging, don't hesitate to @ mention someone for a review.

**Comment by 108adams** [comment]
I find the project dead, as after 1hour of coding and googling I cannot still get the simplest, one page  example from the front page of the project to work. 

welcome our "No files to process" problem.

sorry, no more time to waste, Gatsby's waiting :(

**Comment by michaelkornblum** [comment]
Adam, add a default entry in the layouts object that defines your default handlebars template. I ran into a similar problem some time ago. If memory serves I think that did the trick.

**Comment by ismay** [comment]
@108adams Try our slack channel or the layouts readme. It’s just simple configuration problem, nothing wrong with metalsmith.

**Comment by ismay** [comment]
So I think we can close this, as discussed on the slack channel. The project is alive and actively maintained. If there are still remaining issues maybe we can split them up into more concrete issues or discuss it on slack.
